### PR TITLE
Using latest version of SLDS in RP Resources page

### PR DESCRIPTION
**Pull request by seanpat09**
# Critical Changes

# Changes

# Issues Closed
Updates the RP_Resources page with the latest version of SLDS, replacing the static resource
# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jjbennett** [comment]
Nice to see the last of the --! Can you also remove the sldsscoped from the static resources folder and lurch this PR please?

**Comment by seanpat09** [comment]
**lurch: attach W-028315

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000007fNciAAE" target="_blank">W-028315</a>

**Comment by seanpat09** [comment]
@jjbennett It looks like RP_Getting started wasn't in the StaticResourcesSources folder, so I pulled it out of the static resource, copied it over, and removed the SLDSSCOPED folder

**Comment by seanpat09** [comment]
@jjbennett I did a diff on StaticResourceFolder and the unzipped CumulusStaticResources and found that they are in sync, let me know if you notice any more needed changes!

**Comment by seanpat09** [comment]
@jjbennett got the static resources updated and tested a few links and looks like it's working. Can you review again?

File: src/pages/RP_Resources.page [hunk 1]
```diff
@@ -1,13 +1,13 @@
-<apex:page showHeader="true" sidebar="false" standardStylesheets="false" applyBodyTag="false">
+<apex:page showHeader="true" sidebar="false" standardStylesheets="false" applyBodyTag="false" lightningStyleSheets="true">
```
Comment (jjbennett) [inline src/pages/RP_Resources.page]: Didn't catch this before but is there any reason to enable lightningStyleSheets here? I thought that was for automated styling with non sldsified pages?

Comment (seanpat09) [reply src/pages/RP_Resources.page]: Not that I know of, I think I was just confused by this documentation: https://www.lightningdesignsystem.com/platforms/visualforce/

I can remove the the lightningStyleSheets attribute, it seems like it's redundant when apex:slds is used

**Comment by jjbennett** [comment]
@seanpat09 - decided to do a stare and compare of what is in master and this branch so I put them side by side and noticed that the Getting Started path is not resizing like it did before. Attaching a screen shot with master on the left. Apologies for not catching this earlier too!
screen shot 2019-01-17 at 5 50 01 pm

**Comment by seanpat09** [comment]
@rponti-sforg This is ready for testing!

**Pull request merged**